examples/CloudRetro/README.md

```diff
@@ -0,0 +1,509 @@
+We will be using STUNner to forward our WebRTC media stream.
+
+First, we will deploy STUNner as a turn server, and configure it, then create a configurable Gateway for it, so we can forward an UDP Route through that. And all this inside the cluster. Marvelous.
+
+First of all, create a namespace for stunner:
+
+```console
+kubectl create namespace stunner
+```
+
+#### Installing STUNner gateway operator
+
+This will be a bit tricky and unlogical, but it wil be fixed asap.
+
+Clone the STUNner gateway operator git repo and enter into the root directory:
+
+```console
+git clone https://github.com/l7mp/stunner-gateway-operator.git
+cd stunner-gateway-operator
+```
+
+Install the Kubernetes Gateway CRDs from the official source (these are not part of the STUNner distribution).
+
+```console
+kubectl apply -k "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.3"
+```
+
+Deploy the STUNner Kubernetes Gateway Operator:
+
+```console
+make install
+make deploy
+```
+
+Confirm the operator is running in stunner-gateway namespace:
+
+```console
+kubectl get pods -n stunner-gateway-operator-system
+
+NAME                                                          READY   STATUS    RESTARTS   AGE
+stunner-gateway-operator-controller-manager-65dbf8fb4-hjrjr   2/2     Running   0          42m
+```
+
+#### Configuring the operator
+
+The STUNner operator (partially) implements the official Kubernetes Gateway API, which allows you to interact with STUNner using the convenience of kubectl and declarative YAML configuration.
+
+Create a GatewayClass. This will serve as the root level configuration for your STUNner deployment and specifies the name and the description of the service implemented by the GatewayClass, as well as a Kubernetes resource (the GatewayConfig resource given under the parametersRef) that will define some general parameters for the data-plane implementing the GatewayClass.
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: GatewayClass
+metadata:
+  name: stunner-gatewayclass
+  namespace: stunner
+spec:
+  controllerName: "stunner.l7mp.io/gateway-operator"
+  parametersRef:
+    group: "stunner.l7mp.io"
+    kind: GatewayConfig
+    name: stunner-gatewayconfig
+    namespace: stunner
+  description: "STUNner is a WebRTC ingress gateway for Kubernetes"
+EOF
+```
+
+Next, we specify some important configuration for STUNner, by loading a GatewayConfig custom resource into Kubernetes. Make sure to remember the credentials, these will be needed to connect to the TURN server.
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: stunner.l7mp.io/v1alpha1
+kind: GatewayConfig
+metadata:
+  name: stunner-gatewayconfig
+  namespace: stunner
+spec:
+  stunnerConfig: "stunnerd-configmap"
+  realm: stunner.l7mp.io
+  authType: plaintext
+  userName: "user-1"
+  password: "pass-1"
+EOF
+```
+
+#### Installing STUNner and setting up the gateway
```

Comment:
Again, please refer to the new helm charts: https://github.com/l7mp/stunner-helm